### PR TITLE
update docs to use the new CLI as the default happy path

### PR DESCRIPTION
**Pull request by jeffchuber**
Land after https://github.com/chroma-core/chroma/pull/1032

### CODE REVIEW

File: docs/usage-guide.md [hunk 1]
```diff
@@ -93,9 +93,11 @@ await client.reset() # Empties and completely resets the database. ⚠️ This i
 <Tabs queryString groupId="lang" className="hideTabSwitcher">
 <TabItem value="py" label="Python">
 
-Chroma can also be configured to use an on-disk database, useful for larger data which doesn't fit in memory. To run Chroma in client server mode, run the docker container:
+Chroma can also be configured to use an on-disk database, useful for larger data which doesn't fit in memory. To run Chroma in client server mode:
```
Comment (HammadB) [inline docs/usage-guide.md:96]: I think there are other references to docker we need to rethink as well. For example in auth - https://docs.trychroma.com/usage-guide#basic-authentication

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -29,14 +29,13 @@ results = collection.query(
...
 :::
 
-
 ## Your index resets back to just a few number of records
 
-Users report that they are using Chroma, happily adding data, and then they go to check the `count()` or `query()` and only a single item or a very small fraction of their data is in Chroma. 
+Users report that they are using Chroma, happily adding data, and then they go to check the `count()` or `query()` and only a single item or a very small fraction of their data is in Chroma.
 
-Chroma has 3 clients: `Ephemeral`, `Persistent`, and `Http` (coming soon). The `Ephemeral` and `Persistent` clients should treated as a **singleton**. 
+Chroma has 3 clients: `Ephemeral`, `Persistent`, and `Http` (coming soon). The `Ephemeral` and `Persistent` clients should treated as a **singleton**.
```
Comment (HammadB) [inline docs/troubleshooting.md:36]: Why is this (Coming soon)?

Comment (HammadB) [reply docs/troubleshooting.md:36]: HttpClient exists today

File: docs/usage-guide.md [hunk 3]
```diff
@@ -769,22 +768,26 @@ Supported authentication methods:
 A good security practice is to store the password securely. In the example below we use bcrypt (currently the only supported hash in Chroma server side auth) to hash the plaintext password.
 :::
 
+To generate the password hash, run the following command. Note that you will need to have `htpasswd` installed on your system.
```
Comment (HammadB) [inline docs/usage-guide.md:771]: Can we tell you how to install htpasswd. The reason we used docker here is because it was an assumed dependency

**Review by HammadB — APPROVED** [review]
Please TAL/Address comments but is good for approval. Thanks

File: docs/usage-guide.md (hunk 3 above)
Comment (atroyn) [reply docs/usage-guide.md:771]: Discussed: users configuring auth should know enough to be able to install system utilities on their platform.

**Pull request merged**